syntax = "proto3";

package opentera.protobuf;

message UserDisconnected {

    string user_uuid = 1;
}